README.md

```diff
@@ -33,16 +33,16 @@ the transformations in order to revert them as needed.
 | :book: **[Documentation]**                    | Quickstarts, User and Development Guides, and API Reference.         |
 | :octocat: **[Repository]**                    | The link to the Github Repository of this library.                   |
 | :scroll: **[License]**                        | The entire ecosystem is published under the MIT License.             |
-| :keyboard: **[Development Status]**           | This software is in its Pre-Alpha stage.                             |
+| :keyboard: **[Development Status]**           | This software is in its Alpha stage.                             |
 | [![][Slack Logo] **Community**][Community]    | Join our Slack Workspace for announcements and discussions.          |
 | [![][MyBinder Logo] **Tutorials**][Tutorials] | Run the SDV Tutorials in a Binder environment.                       |
 
 [Website]: https://sdv.dev
 [SDV Blog]: https://sdv.dev/blog
-[Documentation]: https://sdv.dev/SDV
+[Documentation]: https://datacebo.gitbook.io/rdt
```

Comment:
This should change to: https://docs.sdv.dev/RDT

Note: This link isn't live now but it's our intended URL & I'll look to make the switch soon